timeseries.smooth() fails in Python 3

In Py 3
```
>       return y[(window_len-1)/2:-(window_len-1)/2]  # take off repeats on ends
E       TypeError: slice indices must be integers or None or have an __index__ method
```

--- a/CHANGES
+++ b/CHANGES
@@ -2,6 +2,11 @@
  CHANGELOG for numkit
 ======================
 
+2018-08-08      1.1.1
+orbeckst
+
+* fixed numkit.timeseries.smooth() under Python 3 (issue #7)
+
 2018-04-27      1.1.0
 orbeckst, kain88-de
 

--- a/src/numkit/tests/test_timeseries.py
+++ b/src/numkit/tests/test_timeseries.py
@@ -47,5 +47,8 @@ class TestRegularizedFunction(object):
         # assert_almost_equal(np.max(Y), np.max(data[1]))
         # add test for Y data (but has randomness)
 
-
-
+@pytest.mark.parametrize('window', (
+    'flat', 'hanning', 'hamming', 'bartlett', 'blackman'))
+def test_smooth(data, window):
+    a = numkit.timeseries.smooth(data[1], window=window)
+    assert len(a) == len(data[1])

--- a/src/numkit/timeseries.py
+++ b/src/numkit/timeseries.py
@@ -2,6 +2,8 @@
 # Copyright (c) 2010 Oliver Beckstein <orbeckst@gmail.com>
 # Released under the "Modified BSD Licence" (see COPYING).
 
+from __future__ import absolute_import, division
+
 from six.moves import zip as izip
 
 import numpy
@@ -232,7 +234,7 @@ def smooth(x, window_len=11, window='flat'):
 
     s = numpy.r_[x[window_len-1:0:-1], x, x[-1:-window_len:-1]]
     y = numpy.convolve(w/w.sum(), s, mode='valid')
-    return y[(window_len-1)/2:-(window_len-1)/2]  # take off repeats on ends
+    return y[(window_len-1)//2:-(window_len-1)//2]  # take off repeats on ends
 
 def smoothing_window_length(resolution, t):
     """Compute the length of a smooting window of *resolution* time units.

